tests/data/pytest/tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+def pytest_collection_modifyitems(session, config, items):
+    for item in items:
```

Comment:
What is your intent for adding these properties manually??

Reply:
To add these information to the XML file.

Reply:
I couldn't understand the motivation that you want to add these information to the XML files. Could you tell me that?

Reply:
Ops, this is not a test case? Are these configurations for all the test cases?

Reply:
> Ops, this is not a test case? Are these configurations for all the test cases?

Yes it is.

Reply:
TIL: https://stackoverflow.com/questions/34466027/what-is-conftest-py-for-in-pytest